samples/excel/89-preview-apis/document-properties.yaml

```diff
@@ -0,0 +1,118 @@
+order: 6
+id: excel-document-properties
+name: Document properties
+description: Get and set document properties.
+host: EXCEL
+api_set:
+    ExcelApi: 1.7
+script:
+    content: |+
+        $("#get-doc-properties").click(() => tryCatch(getDocProperties));
+        $("#set-get-doc-properties").click(() => tryCatch(setGetDocProperties));
+
+        async function getDocProperties() {
+            await Excel.run(async (context) => {
+                let docProperties = context.workbook.properties;
+
+                // Load the read-only document properties.
+                docProperties.load("author, lastauthor, revisionnumber, creationdate");
+
+                await context.sync();
+
+                console.log("Author  :" + docProperties.author);
+                console.log("Last author  : " + docProperties.lastAuthor);
+                console.log("Revision number : " + docProperties.revisionNumber);
+              
+                OfficeHelpers.UI.notify("Workbook creation date : " + docProperties.creationDate.toDateString());            
+            });
+        }
+
+        async function setGetDocProperties() {
+            await Excel.run(async (context) => {
+                let titleValue = "Excel document properties API";
+                let subjectValue = "Set and get document properties";
+                let keywordsValue = "Set and get operations";
+                let commentsValue = "This is an Excel document properties API code sample";
+                let categoryValue = "Office Add-ins";
+                let managerValue = "John";
+                let companyValue = "Microsoft";
+
+                let docProperties = context.workbook.properties;
+
+                // Set the docoment properties.
```

Comment:
The code here is just creating and writing to properties of a local variable. It's not writing that variable back to `context.workbook.properties`, so it's not really showing how to set the document properties.